computation-slurm/src/main/java/com/powsybl/computation/slurm/SlurmStatusManager.java

```diff
@@ -25,7 +25,7 @@ class SlurmStatusManager {
     private static final String TIME_CMD = "date +\"%Y-%m-%d %H:%M:%S\"";
     private static final String INFO_CORES_CMD = "sinfo -h -o %C";
     private static final String QUEUE_CORES_PER_JOB = "squeue -h --format=\"%C %j\"";
-    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss\n");
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss\n").withZone(ZoneOffset.UTC);
```

Comment:
Should we really use the UTC zone here? Isn't it better to use the system zone?

If this latter option is adopted, the following elements should be changed as:
```java
TIME_CMD = "date +\"%Y-%m-%d %H:%M:%S %Z\"";
DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss zzz\n")
```

Reply:
I personally find it better to always use the UTC timezone in order to be sure to be consistent everywhere.
Your solution however also works.